controllers/encounters.js

```diff
@@ -0,0 +1,19 @@
+const models = require('../models')
+
+const deleteEncounter = async (request, response) => {
```

Comment:
It's unfortunate because this controller actually works correctly but there would be no way a user could ever figure out how to call it.